LightAPRS-hab/example-aprs-config.h

```diff
@@ -0,0 +1,11 @@
+// To use: Rename this file to aprs-config.h
+
+// MUST CHANGE THIS
+#define CALLSIGN "N0CALL" // DON'T FORGET TO CHANGE THIS
```

Comment:
Is this suppose to be #define CALLSIGN "NOCALL"?

Reply:
I think this one won't matter if its N0CALL or NOCALL, as it needs to be changed to the actual callsign regardless.

Reply:
Ah good point

Reply:
I think N0CALL was chosen because it's closer to the format of a valid callsign, but it really shouldn't matter what goes here.